tests/automated_install.sh

```diff
@@ -59,11 +66,18 @@ printf "#### Setup VB's and Bootstrap\n"
 source ./vbox_create.sh
 
 # Ensure we got VM_LIST from vbox_create.sh
-echo "Using CLUSTER_PREFIX=${BACH_CLUSTER_PREFIX} VM_LIST=${VM_LIST}"
+echo "Using CLUSTER_PREFIX=${BACH_CLUSTER_PREFIX}"
+echo "VM LIST"
+for vm in ${VM_LIST[*]}; do
+  echo $vm
+done
 
 download_VM_files || ( echo "############## VBOX CREATE DOWNLOAD VM FILES RETURNED $? ##############" && exit 1 )
 create_bootstrap_VM || ( echo "############## VBOX CREATE BOOTSTRAP VM RETURNED $? ##############" && exit 1 )
 
+echo "VM LIST"
```

Comment:
Another set of `$VM_LIST` echo-ing; again context; is this necessary?

Reply:
removed